docs/installation_linux.rst

```diff
@@ -53,16 +57,16 @@ Poetry is used to manage Python dependencies of LISA.
 
 .. code:: bash
 
-   curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3 -
+   curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3 - --version 1.1.5
 
 After running this, you should see
 ``Add export PATH="/home/YOURUSERNAME/.local/bin:$PATH" to your shell configuration file``
-on the console. Then do
+message on the console. Follow the message and add the necessary exports to ``$HOME/.profile`` file. Then do
 
 .. code:: bash
 
    source $HOME/.profile
-   poetry install
+   make setup
```

Comment:
It may not work on windows.

Reply:
This file is specific to Linux installation. The poetry commands have some flags set and the overall command seemed a bit complex. So, I used this short version instead